src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/transport/action/indexpolicy/TransportIndexPolicyAction.kt

```diff
@@ -59,14 +63,23 @@ class TransportIndexPolicyAction @Inject constructor(
 ) : HandledTransportAction<IndexPolicyRequest, IndexPolicyResponse>(
         IndexPolicyAction.NAME, transportService, actionFilters, ::IndexPolicyRequest
 ) {
+
+    @Volatile lateinit var policy: Policy
```

Comment:
Move this inside doExecute or it'll leak across requests

Reply:
Or just move onto the IndexPolicyHandler class

Reply:
moved to IndexPolicyHandler class